Jenkinsfile

```diff
@@ -5,6 +5,11 @@ pipeline {
   options {
     ansiColor('xterm')
   }
+  environment {
+    PACKAGE = 'kframework'
+    VERSION = '5.0.0'
+    ROOT_URL = 'https://github.com/kframework/k/releases/download/nightly'
```

Comment:
More descriptive names might be good. Eg `PACKAGE_VERSION`, `PACKAGE_ROOT_URL`. Your choice.